internal/backends/postgresql/metadata/registry.go

```diff
@@ -465,11 +468,19 @@ func (r *Registry) collectionCreate(ctx context.Context, p *pgxpool.Pool, dbName
 	list := maps.Values(colls)
 
 	for {
-		tableName = fmt.Sprintf("%s_%08x", strings.ToLower(collectionName), s)
+		tableName = strings.ToLower(collectionName)
 		if strings.HasPrefix(tableName, reservedPrefix) {
 			tableName = "_" + tableName
 		}
 
+		suffixHash := fmt.Sprintf("_%08x", s)
+		if len(tableName) > maxTableNameLength-len(suffixHash) {
+			// trim table name to fit in maxTableNameLength
+			tableName = tableName[0 : maxTableNameLength-len(suffixHash)]
+		}
```

Comment:
```suggestion
		if l :=  maxTableNameLength-len(suffixHash); len(tableName) > l {
			tableName = tableName[:l]
		}
```

Reply:
Oh, I need the same for index names too...

Reply:
thanks!